src/export.js

```diff
@@ -21,6 +21,11 @@ export default function exportQUnit (QUnit) {
 
```

Comment:
Do you know what it is about `module.exports = QUnit;` that doesn't work for ESM compat layers? It seems odd to assign `module.exports = QUnit;` but then (seemingly redundantly) also assign `module.exports.test = QUnit.test;` to itself. If the transformation layers are based on static analysis, that would explain it, but I thought they were runtime based?

Reply:
maybe. as I dug in to this, I think it'd be better to have actual separate builds -- and we're already in a good spot since it seems QUnit is written in ESM -- so we'd just emit a new bundle and wire up `package.json#exports` so that the import/require/etc are routed correctly.